### PR TITLE
Added VB/VBA example

### PR DESCRIPTION
**Pull request by kotliary**
I've added VBA function and example code. Successfully used it in Outlook to push message to Gotify when a new mail arrive from predefined list of senders.

### CODE REVIEW

File: docs/more-pushmsg.md [hunk 1]
```diff
@@ -193,6 +193,77 @@ public class GotifyClient {
...
+
+' ----- copied from ExcelUtils.
+' ----- from: https://github.com/makah/ExcelHelper
+'
+' Generate a string using .NET format, i.e. {0}, {1}, {2} ...
+' @param In strValue as String: A composite format string that includes one or more format items
+' @param In arrParames as Variant: Zero or more objects to format.
+' @return as String: A copy of format in which the format items have been replaced by the string representations of the corresponding arguments.
+' @example: Debug.Print StringFormat("My name is {0} {1}. Hey!", "Mauricio", "Arieira")
+Public Function StringFormat(ByVal strValue As String, ParamArray arrParames() As Variant) As String
```
Comment (jmattheis) [inline docs/more-pushmsg.md:230]: Hey, thanks for your contribution! This code seems pretty bloaty with all the comments and extra string format function. Also, I think it is enough to describe one way to send a message. Could you remove the string format function and the extra information about sending with json?

Comment (kotliary) [reply docs/more-pushmsg.md]: Thank you for the comment. I cleaned up the code by your recommendations. I hope it's okay to leave the link to the code origin in the comment.

**Review by jmattheis — APPROVED** [review]
Thanks.

**Comment by kotliary** [comment]
I've corrected the GOTIFY_URL in the new commit. Do I have to do another pull request? Thanks.

**Comment by jmattheis** [comment]
The url doesn't really matter (:

**Comment by kotliary** [comment]
It actually does. I'd really like to change it. ;)

**Comment by jmattheis** [comment]
Alright, I'll change it.

**Pull request merged**